lib/tasks/migrations/migrations.rake

```diff
@@ -74,6 +74,43 @@ namespace :migrations do
     exit has_encountred_issue
   end
 
+  task :rooms, [] => :environment do |_task, _args|
```

Comment:
If we're enabling partitioning (range) on users and since on v2 a user will have at least one room we may need partitioning here either,

Reply:
Why would we enable partitioning? I saw that in users migration

Reply:
I believe we enabled that task because it's expected to migrate a large number of users,
For the same reason the number of rooms is expected to be even higher,